bus-mapping/src/evm/opcodes.rs

```diff
@@ -530,10 +546,108 @@ fn dummy_gen_call_ops(
         }
         // 3. Call to account with non-empty code.
         (_, false) => Ok(vec![exec_step]),
     }
 }
+
+fn dummy_gen_create_ops(
+    state: &mut CircuitInputStateRef,
+    geth_steps: &[GethExecStep],
+) -> Result<Vec<ExecStep>, Error> {
+    let geth_step = &geth_steps[0];
+    let mut exec_step = state.new_step(geth_step)?;
+
+    let tx_id = state.tx_ctx.id();
+    let call = state.parse_call(geth_step)?;
+
+    // Increase caller's nonce
+    let nonce_prev = state.sdb.increase_nonce(&call.caller_address);
+    state.push_op_reversible(
+        &mut exec_step,
+        RW::WRITE,
+        AccountOp {
+            address: call.caller_address,
+            field: AccountField::Nonce,
+            value: (nonce_prev + 1).into(),
+            value_prev: nonce_prev.into(),
+        },
+    )?;
+
+    // Add callee into access list
+    let is_warm_prev = !state.sdb.add_account_to_access_list(call.address);
+    state.push_op_reversible(
+        &mut exec_step,
+        RW::WRITE,
+        TxAccessListAccountOp {
+            tx_id,
+            address: call.address,
+            is_warm: true,
+            is_warm_prev,
+        },
+    )?;
+
+    state.push_call(call.clone(), geth_step);
+
+    // Increase callee's nonce
+    let nonce_prev = state.sdb.increase_nonce(&call.address);
+    debug_assert!(nonce_prev == 0);
+    state.push_op_reversible(
+        &mut exec_step,
+        RW::WRITE,
+        AccountOp {
+            address: call.address,
+            field: AccountField::Nonce,
+            value: 1.into(),
+            value_prev: 0.into(),
+        },
+    )?;
+
+    let (found, caller_account) = state.sdb.get_account_mut(&call.caller_address);
+    if !found {
+        return Err(Error::AccountNotFound(call.caller_address));
+    }
+    let caller_balance_prev = caller_account.balance;
+    let caller_balance = caller_account.balance - call.value;
+    caller_account.balance = caller_balance;
+    state.push_op_reversible(
+        &mut exec_step,
+        RW::WRITE,
+        AccountOp {
+            address: call.caller_address,
+            field: AccountField::Balance,
+            value: caller_balance,
+            value_prev: caller_balance_prev,
+        },
+    )?;
+
+    let (found, callee_account) = state.sdb.get_account_mut(&call.address);
+    if !found {
+        return Err(Error::AccountNotFound(call.address));
+    }
+    let callee_balance_prev = callee_account.balance;
+    let callee_balance = callee_account.balance + call.value;
+    callee_account.balance = callee_balance;
+    state.push_op_reversible(
+        &mut exec_step,
+        RW::WRITE,
+        AccountOp {
+            address: call.address,
+            field: AccountField::Balance,
+            value: callee_balance,
+            value_prev: callee_balance_prev,
+        },
+    )?;
+
+    if call.code_hash.to_fixed_bytes() == *EMPTY_HASH {
+        // 1. Create with empty initcode.
+        state.handle_return(geth_step)?;
```

Comment:
This seems to be the same case as `CALL` without code discussed at https://github.com/appliedzkp/zkevm-specs/issues/146

I'm just mentioning it in order to take it into account once we address #146 :)